public/stylesheets/custom.css

```diff
@@ -0,0 +1,5 @@
+@media (min-width: 980px) {
+  .upper-header {
+    padding-top: 50px;
```

Comment:
In all honesty the better solution by far would be for me to swap out Bootstrap 2 for Bootstrap 3